src/data/news/software_releases.yml

```diff
@@ -1,3 +1,39 @@
+- 
+  Title: "Bison Relay Release v0.1.7 RC1"
+  Permalink: "https://github.com/companyzero/bisonrelay/releases/tag/v0.1.7-rc1"
+  Params:
+    sortDate: "2023-04-21 20:16:18"
+    icon: "releaseUpdates.svg"
+-
+  Title: "Decred Release v1.7.7"
+  Permalink: "https://github.com/decred/decred-binaries/releases/tag/v1.7.7"
+  Params:
+    sortDate: "2023-04-17 16:00:22"
+    icon: "releaseUpdates.svg"
+- 
+  Title: "DCRDEX Release v0.6.0"
+  Permalink: "https://github.com/decred/dcrdex/releases/tag/v0.6.0"
+  Params:
+    sortDate: "2023-04-17 04:29:48"
+    icon: "releaseUpdates.svg"
+- 
+  Title: "Bison Relay Release v0.1.6 RC1"
```

Comment:
Thanks for laying it out, I removed the RCs.

For the record, my motivation to add them was to show on the News page that BR is evolving quickly and releases are ticking ("heartbeat" effect that shows "X is alive!"). Secondly, to provide more up-to-date download links (https://bisonrelay.org/download/ still at v0.1.4). If the current version we recommend to end users is indeed v0.1.4 then I was trying to solve a non-existent problem. Otherwise, I was trying to solve it in a wrong place (News page) :)

Reply:
the BR site should point to the latest non-RC release, so it should be 0.1.5 at the moment